app/digests/tests/test_digest_api_filtering.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict
+
+from django.conf import settings
+from django.test.client import Client
+import pytest
+
+from digests.models import Digest
+
+DIGESTS_URL = '/digests'
+
+
+@pytest.mark.django_db
+def test_can_filter_by_article_id(can_preview_header: Dict,
+                                  client: Client,
+                                  preview_digest: Digest,
+                                  digest_json: Dict,
+                                  digest_related_content_json: Dict):
+    article_id = digest_related_content_json[0]['id']
+    response = client.get(f'{DIGESTS_URL}?article={article_id}',
+                          **{'ACCEPT': settings.DIGEST_CONTENT_TYPE},
+                          **can_preview_header)
+    assert response.data['total'] == 1
+    assert response.data['items'][0]['id'] == preview_digest.id
+    assert response.data['items'][0]['relatedContent'][0]['id'] == article_id
+
+
+@pytest.mark.django_db
+def test_will_return_empty_if_article_id_not_found(can_preview_header: Dict,
+                                                   client: Client,
+                                                   preview_digest: Digest,
+                                                   digest_json: Dict):
+    article_id = '123456'
+    response = client.get(f'{DIGESTS_URL}?article={article_id}',
```

Comment:
This is why we need to design the API first. We already have a standard for doing this: `?containing []=article/1234`.

Reply:
Yes not being a collection we need to say something that points the filter to the right field `relatedContent`, could still benefit from a wider standard

Reply:
It was an additional requirement given after most of it was already implemented.

I can change the required param, but `containing[]` means nothing to `django` in realation to `JSONField`s by default, it not a colleciton, would have to have a custom implementation.